company-profiles/willings-inc.md

```diff
@@ -0,0 +1,44 @@
+# Willings Inc
+
+## Company blurb
+Enhancing the Presence of Japanese Companies in the Digital Society
+
+Established : 11th January 2018
+
+## Company size
+
+2-10 employees
+
+Source : https://www.linkedin.com/company/willingsinc/about/
+
+## Remote status
+
+Recruitment/Empowerment support of advanced IT engineers from Indian Institutes of Technology “GALK” [13-ユ-309586]
```

Comment:
@sangdeepganvir Here is another India reference, can you please fix. Remote status should explain how the company supports remote employees or what they offer for this.

Reply:
@dougaitken please review it.